### ISSUE TITLE
Check the ability to use feed authenticated URLs

### ISSUE THREAD

**Issue by dannysale** [a-chacon/chaski-app#1]
I love the app, but couldn't find a way to mark things read, and get the old stuff out of the way. I suspect it will be addressed in a future release?

OPML import went well, but I couldn't add a single feed for some reason, even when trying to subscribe to the default Chaski link. It just says it couldn't find anything

I'm using the Windows version, BTW.

Please keep up the great work!

**Comment by a-chacon** [comment]
Hey! Thanks for the feedback. The mark as read feature should be work as hide too. Here is a demo: 

https://github.com/user-attachments/assets/32e1247a-dada-4878-b7aa-894de3ab5169

Please, can you confirm that this is the behavior you have? I just notice that the page "ALL" does not hide the read entries, but the other pages like folder, feed or today does.

The second thing, mm the chaski page don't have any feed yet (I planned to add one for release the updates). Here is a demo working: 

https://github.com/user-attachments/assets/749d8a1e-7950-4abc-aff1-3ac173b39926

Sometimes some feeds cannot be auto-discovered and should show the message you see. 

Thanks again!

**Comment by dannysale** [comment]
Yes, that works! But I think what I'm having a problem with is that I subscribe to a large number of feeds, and need a "Mark ALL read" option that would let me mark everything in a folder as read once I've glanced over it. Otherwise, I'm having to do it on every entry, and that would take quite a bit of time.

**Comment by dannysale** [comment]
Hey, I just noticed the "Mark Read on Hover" feature, and that should be good enough for me. I do have an RSS URL that I can't subscribe to, though. It's in this format:

http://username:password@bt.dimeadozen.org/rss/

I'm not sure if you're aware of sites that feature the kind of URL that contains login information.

**Comment by a-chacon** [comment]
Hey! not, I didn't know there is URLs with username and password, I will check this. And for the "Mark All read" I think I need to add this by folder and in the today page too. Thank you for your comments, I think this issue will be about the authenticated URls

**Comment by dannysale** [comment]
Authentication would be ideal. And yeah, "Mark All Read" is pretty standard. Doing it will hovering isn't bad, but when I import my OPML, there is just way too much stuff to go through one-by-one.

**Comment by a-chacon** [comment]
@dannysale thanks, I already create a new issue for the mark all read feature here: https://github.com/a-chacon/chaski-app/issues/2#issue-2759155863

I will try to solve this asap. Thanks again and please any Issue, Feedback or opinion just write to me, and it would be nice if you share the project, so more people can see it;)

**Comment by dannysale** [comment]
I will!
